packages/flutter/lib/src/widgets/sliver_tree.dart

```diff
@@ -0,0 +1,963 @@
+    this.findChildIndexCallback,
+    this.animationStyle,
+    this.traversalOrder = SliverTreeTraversalOrder.depthFirst,
+    this.indentation = SliverTreeIndentationType.standard,
+  });
+
+  /// The list of [SliverTreeNode]s that may be displayed in the [SliverTreeList].
+  ///
+  /// Beyond root nodes, whether or not a given [SliverTreeNode] is displayed
+  /// depends on the [SliverTreeNode.isExpanded] value of its parent. The
+  /// [SliverTreeList] will set the [SliverTreeNode.parent] and
+  /// [SliverTreeNode.depth] as nodes are built on demand to ensure the
+  /// integrity of the tree.
+  final List<SliverTreeNode<T>> tree;
+
+  /// Called to build and entry of the [SliverTreeList] for the given node.
+  ///
+  /// By default, if this is unset, the [SliverTreeList.defaultTreeNodeBuilder]
+  /// is used.
+  final SliverTreeNodeBuilder treeNodeBuilder;
+
+  /// Called to calculate the extent of the widget built for the given
+  /// [SliverTreeNode].
+  ///
+  /// By default, if this is unset, the
+  /// [SliverTreeList.defaultTreeRowExtentBuilder] is used.
+  ///
+  /// See also:
+  ///
+  ///   * [SliverVariedExtentList.itemExtentBuilder], a very similar method that
+  ///     allows users to dynamically compute extents on demand.
+  final SliverTreeRowExtentBuilder treeRowExtentBuilder;
+
+  /// If provided, the controller can be used to expand and collapse
+  /// [SliverTreeNode]s, or lookup information about the current state of the
+  /// [SliverTreeList].
+  final SliverTreeController? controller;
+
+  /// Called when a [SliverTreeNode] expands or collapses.
+  ///
+  /// This will not be called if a [SliverTreeNode] does not have any children.
+  final SliverTreeNodeCallback? onNodeToggle;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.addAutomaticKeepAlives}
+  final bool addAutomaticKeepAlives;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.addRepaintBoundaries}
+  final bool addRepaintBoundaries;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.addSemanticIndexes}
+  final bool addSemanticIndexes;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.semanticIndexCallback}
+  final SemanticIndexCallback semanticIndexCallback;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.semanticIndexOffset}
+  final int semanticIndexOffset;
+
+  /// {@macro flutter.widgets.SliverChildBuilderDelegate.findChildIndexCallback}
+  final int? Function(Key)? findChildIndexCallback;
+
+  /// Used to override the toggle animation's curve and duration.
+  ///
+  /// If [AnimationStyle.duration] is provided, it will be used to override
+  /// the [SliverTreeList.defaultAnimationDuration], which is 150
+  /// milliseconds.
+  ///
+  /// If [AnimationStyle.curve] is provided, it will be used to override
+  /// the [SliverTreeList.defaultAnimationCurve], which is [Curves.linear].
+  ///
+  /// To disable the tree animation, use [AnimationStyle.noAnimation].
+  final AnimationStyle? animationStyle;
```

Comment:
`animationStyle` could be name `toggleAnimationStyle`, which makes it easier to understand what type of animation this property customizes. In this case it used for toggle animation based on the docs.

E.g. [expansionAnimationStyle](https://main-api.flutter.dev/flutter/material/ExpansionTile/expansionAnimationStyle.html) used to customize `ExpansionTile` expand/collapse animation.